Dockerfile.dev

```diff
@@ -1,4 +1,4 @@
-FROM centos/nodejs-6-centos7:latest
+FROM centos/nodejs-4-centos7:latest
```

Comment:
any reason to roll it back to nodejs4?

Reply:
I had to roll back to 4 to get the dev image to work, someone bumped the dockerfile for dev to 6 but it is not compatible with the commands to be run such as ```scl enable rh-nodejs4``` and I recall the last time I checked there was no support for node 6 with this command yet so it was easier to roll back to get the dev environment up and running

Reply:
I see now it looks like 6 is supported so i can try that out. This should not affect the rhel image built from the pr though right?